openmls/src/schedule/pprf/prefix.rs

```diff
@@ -0,0 +1,148 @@
+//! This module defines a trait for prefixes to instantiate a PPRF depending on
```

Comment:
The draft only defines `LeafIndex(ComponentID)` to index the tree. Can you describe a little what the other types are for?